lib/utils/cli.go

```diff
@@ -102,14 +111,22 @@ func InitLogger(purpose LoggingPurpose, level slog.Level, opts ...LoggerOption)
 	// then discard all log output.
 	if purpose == LoggingForCLI && level > slog.LevelDebug {
 		slog.SetDefault(slog.New(logutils.DiscardHandler{}))
-		return
+		return nil
+	}
+
+	var output string
+	if o.osLogSubsystem != "" {
+		output = logutils.LogOutputOSLog
 	}
 
-	logutils.Initialize(logutils.Config{
-		Severity:     level.String(),
-		Format:       o.format,
-		EnableColors: IsTerminal(os.Stderr),
+	_, _, err := logutils.Initialize(logutils.Config{
+		Severity:       level.String(),
+		Format:         o.format,
+		EnableColors:   IsTerminal(os.Stderr),
```

Comment:
do colors matter for oslog?

Reply:
os_log doesn't support colors and this option is effectively ignored for os_log.

https://github.com/gravitational/teleport/blob/60080b069ff464804be3b6a6e4601b45034d9ed1/lib/utils/log/slog_text_handler_darwin.go#L28-L34